src/acquisition/covid_hosp/common/database.py

```diff
@@ -12,17 +11,17 @@
 # first party
 import delphi.operations.secrets as secrets
 from delphi.epidata.common.logger import get_structured_logger
-
-Columndef = namedtuple("Columndef", "csv_name sql_name dtype")
+from delphi.epidata.common.covid_hosp.covid_hosp_schema_io import CovidHospSomething
 
 class Database:
 
+  DATASET_NAME = None
+  # Defined outside the constructor as this does not require a DB connection
+  METADATA_ID = CovidHospSomething().get_metadata_id(DATASET_NAME) if DATASET_NAME is not None else None
```

Comment:
Isn't set alongside the other fields in the constructor, since it's only used by `Network` - and that doesn't require an actual `Database` instance, just the metadata ID string.